Microsoft.Toolkit.Uwp.UI/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,205 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should we just make this StringExtensions.Vector.cs or something in case we have others in the future we can start splitting them out? Also looks like mv didn't take? ☹

Reply:
I'm thinking we could just leave this as is for now until we don't have other extensions, since it's just an internal detail anyway? And if we do add more extensions in the future we can always just `git mv` this file anyway 🤔